### PR TITLE
Added Introduction

### PR DESCRIPTION
**Pull request by SirBinky**
Added Introduction and a few set up points linking to nightscouts already good setup guides.

### CODE REVIEW

File: docs/docs/walkthrough/phase-1/visualization.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
-# Visualization and Monitoring
+ # Visualization and Monitoring
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/visualization.md]: Why the extra space?

File: docs/docs/walkthrough/phase-1/visualization.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+ [Nightscout](http://nightscout.info) is the recommended way to visualize your OpenAPS closed loop. Even if you don't choose to share your Nightscout instance with another person, it will be helpful for you to visualize what the loop is doing; what it's been doing; plus generate helpful reports for understanding your data and also allowing you to customize watchfaces with your OpenAPS data. This provides a visual alternative to SSHing into your RaspberryPI or loop system and looking through log files. 
+ 
+Please note if you have decided to connect your CGM receiver directly to the raspberry pi to create a system that works offline too then nightscout is not necessary as you will receive your CGM data directly. Although you will not see what OpenAPS has recommended and how that impacted on blood glucose levels.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/visualization.md]: We really don't want to encourage anyone to set up OpenAPS without a good method of visualizing what the loop is doing

File: docs/docs/walkthrough/phase-1/visualization.md (hunk 1 above)
Comment (SirBinky) [inline docs/docs/walkthrough/phase-1/visualization.md]: I cut and paste the whole top part of the doc into a word processor to spell check what I did. I didn't notice the extra space when i moved it back.

File: docs/docs/walkthrough/phase-1/visualization.md (hunk 2 above)
Comment (SirBinky) [inline docs/docs/walkthrough/phase-1/visualization.md]: Ah ok, so best to remove 11th line entirely then. Shall I do that and resubmit?

Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/visualization.md]: If you just do another commit to fix that line (and the space one), it will automatically update the PR.
